desktop/apps/auction2/client.js

```diff
@@ -62,22 +62,29 @@ render(
 
 store.dispatch(actions.fetchArtworks())
 
-// 1 second delay for checkbox selections
-let timer = null
+// scroll up if you select a checkbox or sort
 function delayedScroll() {
-  clearTimeout(timer)
-  timer = setTimeout($('html,body').animate( { scrollTop: 0 }, 400), 1000)
+  $('html,body').animate( {
+    scrollTop: $('.auction2-my-active-bids').offset().top - $('.mlh-navbar').height()
+  }, 400)
 }
 
+$('body').on('click', '.artsy-checkbox', () => delayedScroll())
+$('body').on('click', '.bordered-pulldown-options a', (e) => {
+  e.preventDefault()
+  $('.bordered-pulldown-options').hidehover()
+  delayedScroll()
```

Comment:
Oh, good catch. I took out the delay because I couldn't seem to get it working :( I'll fix this naming and add it back if we need it.